tests/onepay/test_transaction.py

```diff
@@ -45,26 +52,65 @@ def test_validate_create(self):
             Transaction.create(self.get_valid_cart(), Channel.MOBILE)
 
     def test_raise_error_response_create_transaction(self):
-        onepay.api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
-        onepay.shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
-
         with requests_mock.Mocker() as m:
             m.register_uri("POST", re.compile("/sendtransaction"), text="{\"response_code\": \"ERROR\", \"description\": \"ERROR\"}")
             
             with self.assertRaisesRegex(TransactionCreateError, "ERROR : ERROR"):
                 response = Transaction.create(self.get_valid_cart(), Channel.WEB)
 
-    def test_create_transaction(self):
-        onepay.api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
-        onepay.shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
+    def test_create_transaction_global_options(self):
         response = Transaction.create(self.get_valid_cart(), Channel.WEB)
 
         self.assertIsNotNone(response)
+        self.assertIsNotNone(response)
```

Comment:
I think this is duplicated (or missing a method call?)

Reply:
@eugenioLopezRamos fixed! Thanks.